pgcli/main.py

```diff
@@ -729,11 +734,15 @@ def get_prompt(self, string):
         help='Use DSN configured into the [alias_dsn] section of pgclirc file.')
 @click.option('-R', '--row-limit', default=None, envvar='PGROWLIMIT', type=click.INT,
         help='Set threshold for row limit prompt. Use 0 to disable prompt.')
+@click.option('--less-chatty', 'less_chatty', is_flag=True,
+        default=None,
+        help='Skip intro on startup and goodbye on exit.')
```

Comment:
Wording taken verbatim from the default pgclirc file.

One alternative is "Suppress the startup message", from issue #626, but I went with the more verbose option.